### ISSUE TITLE
why my result is so bad, is there anything i can do to imporve it

### ISSUE THREAD

**Issue by yier2333** [YBIGTA/pytorch-hair-segmentation#63]
![image](https://user-images.githubusercontent.com/10252024/205838054-a314fd0c-c61b-4bec-90a9-3b433cb11762.png)

**Comment by Demste** [comment]
if you still wonder my training configs and result

![Image](https://github.com/user-attachments/assets/18e12e8f-1814-464b-bbc2-933b9be4d748)

!python3 /content/pytorch-hair-segmentation/main.py \
  --networks mobilenet \
  --dataset figaro \
  --data_dir /content/Figaro1k \
  --scheduler ReduceLROnPlateau \
  --batch_size 8 \
  --epochs 50 \
  --lr 1e-3 \
  --num_workers 4 \
  --optimizer adam \
  --img_size 512 \
  --momentum 0.5 \
  --ignite True
